packages/firebase_auth/ios/Classes/FirebaseAuthPlugin.m

```diff
@@ -111,8 +111,9 @@ - (void)handleMethodCall:(FlutterMethodCall *)call result:(FlutterResult)result
       [self sendResult:result forUser:nil error:nil];
     }
   } else if ([@"getIdToken" isEqualToString:call.method]) {
+    BOOL forceRefresh = call.arguments[@"refresh"];
```

Comment:
pretty sure this value will be an object, so you'll need to do something like:
```
BOOL forceRefresh = false;
NSNumber *refresh = call.arguments[@"refresh"];
if (refresh != nil && refresh != NSNull) {
forceRefresh = refresh.boolValue;
}

Reply:
Charlies right. I believe you can tighten it up to 
```
BOOL forceRefresh = false;
NSNumber *refresh = call.arguments[@"refresh"];
if (refresh) {
   forceRefresh = [refresh boolValue];
}
```